setup.cfg

```diff
@@ -45,7 +45,7 @@ install_requires =
 dev =
     coverage [toml]
     pytest
-    pytest-cookies
+    pyprojroot
```

Comment:
I am not familiar with pyprojroot. I was originally thinking of replacing 'pytest-cookies'  with 'pytest-copie'. See https://github.com/12rambau/pytest-copie

Reply:
`pyprojroot` is a simple but very helpful package to easily refer to paths relative to the project root. For the tests specifically, I used it to guarantee that the source refers to the root of the project, regardless of where the tests are called from (which can sometimes cause issues with these things.)

Using `pytest-copie` would be nice, but it doesn't a session-scoped fixture such as the one that `pytest-cookies` does. That means we end up with a whole bunch of `ScopeMismatch` errors if we do a 'naive' replacement:

```python
ERROR tests/test_project.py::test_pytest - Failed: ScopeMismatch: You tried to access the function scoped fixture copie with a session scoped request object. Requesting fixture stack:
ERROR tests/test_project.py::test_coverage - Failed: ScopeMismatch: You tried to access the function scoped fixture copie with a session scoped request object. Requesting fixture stack:
ERROR tests/test_project.py::test_tox - Failed: ScopeMismatch: You tried to access the function scoped fixture copie with a session scoped request object. Requesting fixture stack:
ERROR tests/test_project.py::test_subpackage - Failed: ScopeMismatch: You tried to access the function scoped fixture copie with a session scoped request object. Requesting fixture stack:
ERROR tests/test_project.py::test_generate_api_docs - Failed: ScopeMismatch: You tried to access the function scoped fixture copie with a session scoped request object. Requesting fixture stack:
ERROR tests/test_project.py::test_coverage_api_docs - Failed: ScopeMismatch: You tried to access the function scoped fixture copie with a session scoped request object. Requesting fixture stack:
ERROR tests/test_project.py::test_doctest_api_docs - Failed: ScopeMismatch: You tried to access the function scoped fixture copie with a session scoped request object. Requesting fixture stack:
ERROR tests/test_project.py::test_ruff_check - Failed: ScopeMismatch: You tried to access the function scoped fixture copie with a session scoped request object. Requesting fixture stack:
ERROR tests/test_project.py::test_bumpversion - Failed: ScopeMismatch: You tried to access the function scoped fixture copie with a session scoped request object. Requesting fixture stack:
```

Looking at the `pytest-cookies` [code](https://github.com/hackebrot/pytest-cookies/blob/28492af5bba1da3a2bed19389f96584e51c1e08f/src/pytest_cookies/plugin.py#L159), it seems like it should be an easy contribution to add a similar session scoped version. We should look into that!